packages/reporter/src/lib/scroller.ts

```diff
@@ -91,8 +178,13 @@ export class Scroller {
   _isFullyVisible (element: HTMLElement) {
     if (!this._container) return false
 
-    return element.offsetTop - this._container.scrollTop > 0
-           && this._container.scrollTop > this._aboveBottom(element)
+    const elementOffsetTop = element.offsetTop
+    const elementClientHeight = element.clientHeight
+    const containerScrollTop = this._container.scrollTop
+    const containerClientHeight = this._container.clientHeight
```

Comment:
Call dom references once. This will only fallback is intersection observer data isn't available.